layers/descriptor_validation.cpp

```diff
@@ -3308,3 +3396,913 @@ bool CoreChecks::VerifyWriteUpdateContents(const DescriptorSet *dest_set, const
     // All checks passed so update contents are good
     return true;
 }
+
+bool CoreChecks::PreCallValidateCmdSetDescriptorBufferOffsetsEXT(VkCommandBuffer commandBuffer,
+                                                                 VkPipelineBindPoint pipelineBindPoint, VkPipelineLayout layout,
+                                                                 uint32_t firstSet, uint32_t setCount,
+                                                                 const uint32_t *pBufferIndices,
+                                                                 const VkDeviceSize *pOffsets) const {
+    auto cb_state = GetRead<CMD_BUFFER_STATE>(commandBuffer);
+    auto pipeline_layout = Get<PIPELINE_LAYOUT_STATE>(layout);
+    assert(cb_state);
+    assert(pipeline_layout);
+
+    bool skip = false;
+
+    if (!enabled_features.descriptor_buffer_features.descriptorBuffer) {
+        skip |= LogError(device, "VUID-vkCmdSetDescriptorBufferOffsetsEXT-None-08060",
+                         "vkCmdSetDescriptorBufferOffsetsEXT(): The descriptorBuffer feature "
+                         "must: be enabled.");
```

Comment:
nit:  not asciidoctor

```suggestion
                         "must be enabled.");
```

few other spots below too